.deploy/dev-gcp.json

```diff
@@ -0,0 +1,6 @@
+{
+  "ingresses": ["https://familie-dokument.dev-gcp.nais.io"],
```

Comment:
Hvordan får vi gitt søknadsdialogen (som er på nav.no) tilgang til å kalle denne urlen? må isåfall ha noe CORS på plass, kanskje?

Reply:
True! I'll add CORS configurations. Do I need anything else to enable the access?

Reply:
We need `nav.no` ingress